### ISSUE TITLE
[sonatype-nexus] Wrong identation in ingress.yaml

### ISSUE THREAD

**Issue by SakaSun** [Oteemo/charts#43]
I was trying to configure additional ingress rules, but the identation applied on `.Values.ingress.rules` in `ingress.yaml` fails at yaml validation.

**Comment by rjkernick** [comment]
Can you provide the overrides you were applying to the chart while you were seeing an error?

**Comment by SakaSun** [comment]
Sure. In my custom `values.yaml`:

```
  rules:
  - host: hosted.docker.example.com
    http:
      paths:
      - path: /repository/docker-releases/
        backend:
          serviceName: nexus-sonatype-nexus
          servicePort: 8081
  - host: hub.docker.example.com
    http:
      paths:
      - path: /repository/docker-proxy/
        backend:
          serviceName: nexus-sonatype-nexus
          servicePort: 8081
```

**Comment by SakaSun** [comment]
It should be idented with 4 spaces instead of 2 in `ingress.yaml`.

**Comment by famenc** [comment]
> 
> 
> It should be indented with 4 spaces instead of 2 in `ingress.yaml`.

I was getting the same error, and this worked (changing the indent to 4 in .Values.ingress.rules

**Comment by rjkernick** [comment]
Issue should be fixed now that #52 is merged
